felix/rules/static.go

```diff
@@ -1121,16 +1121,22 @@ func (r *DefaultRuleRenderer) StaticBPFModeRawChains(ipVersion uint8,
 	}
 
 	rawRules = append(rawRules,
-		Rule{
-			Match:  Match().NotDestAddrType(AddrTypeLocal),
-			Action: GotoAction{Target: ChainRawUntrackedFlows},
-		},
 		Rule{
 			// Return, i.e. no-op, if bypass mark is not set.
```

Comment:
whyis this the first rule now? Does it make any difference?

Reply:
Because this rule uses `0xffffffff` for mask and we change the mark the newly added rule. If we change the mark in with `SKipFIB` then this rule always fails and for that `doNotTrack` policies won't work. 
At the same time, the next two rules are relevant so I put them near each others.